packages/front-end/pages/experiments/index.tsx

```diff
@@ -294,6 +295,7 @@ const ExperimentsPage = (): React.ReactElement => {
   }
 
   const hasExperiments = experiments.length > 0;
+  const hasDescription = experiments.some((e) => e.description);
```

Comment:
I like this logic - I think we should apply it to some of the other columns (tags and project, namely).